src/Gemfile

```diff
@@ -24,8 +24,8 @@ group :development do
 end
 
 group :test do
-  gem 'faker'
-  gem 'shoulda-matchers'
+  gem 'faker', '~> 2.11.0'
+  gem 'shoulda-matchers', '~> 4.3'
```

Comment:
[Q] why do you need to fix the version here? it's already locked in `gemfile.lock`?